.ci/scripts/diff_c10_mirror_with_pytorch.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+$(dirname "${BASH_SOURCE[0]}")/compare_dirs.sh runtime/core/portable_type/c10/c10 pytorch/c10
```

Comment:
Why do we need this script for just a single line? Can we just use the line directly in `check_c10_sync.sh`?

If not, I guess we want to either `exec` this or `set -eu` up top